scripts/aad/aad-app-reg.sh

```diff
@@ -571,7 +571,7 @@ JSON
 )
 
   # Is the Swagger UI app already registered?
```

Comment:
Does this create a new AAD App the first time it runs? Do we need to tell people to delete the old Swagger AAD App